apps/meteor/client/views/room/composer/ComposerSelectMessages.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, ButtonGroup } from '@rocket.chat/fuselage';
+import { MessageFooterCallout, MessageFooterCalloutContent } from '@rocket.chat/ui-composer';
+import type { ReactElement } from 'react';
+import React, { useContext } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import type { ComposerMessageProps } from './ComposerMessage';
+import { useCountSelected, useClearSelection, SelectedMessageContext } from '../MessageList/contexts/SelectedMessagesContext';
+
+const ComposerSelectMessages = ({ onGetMore }: ComposerMessageProps): ReactElement => {
```

Comment:
Naming here could use some improvement too.

```suggestion
const ComposerSelectMessages = ({ onClickSelectAll }: ComposerMessageProps): ReactElement => {
```

Or maybe something different, just a suggestion.